drape_frontend/apply_feature_functors.cpp

```diff
@@ -390,11 +389,52 @@ void BaseApplyFeature::ExtractCaptionParams(CaptionDefProto const * primaryProto
   }
 }
 
-ApplyPointFeature::ApplyPointFeature(TileKey const & tileKey, TInsertShapeFn const & insertShape, FeatureID const & id,
-                                     uint8_t rank, CaptionDescription const & captions, float posZ)
-  : TBase(tileKey, insertShape, id, rank, captions)
+double BaseApplyFeature::PriorityToDepth(int const priority, drule::rule_type_t const ruleType, double const areaDepth) const
```

Comment:
nit: It looks very safe, but we usually don't write const for integral types in input function params by value.
Unusual syntax overhead imo.

Reply:
fixed